packages/amplication-server/src/core/resource/sampleResource.ts

```diff
@@ -16,7 +16,8 @@ import { ResourceCreateInput } from './dto';
 export const SAMPLE_SERVICE_DATA: ResourceCreateInput = {
   description: 'Sample Service for task management',
   name: 'My sample service',
-  resourceType: EnumResourceType.Service
+  resourceType: EnumResourceType.Service,
+  project: { connect: { id: 'exampleProjectId' } }
```

Comment:
this will not work... 
I am not sure if we are using this const at all. it we are please test and fix - if we are not, please remove it

Reply:
@yuval-hazaz we are using only in `resource.service.spec.ts`